### ISSUE TITLE
Don't hog all my sundays

### ISSUE THREAD

**Issue by joelhellman** [RasmusRossetti/TimeLockBeta#7]
There is a person that always book the sundays 15-18, when I also want to wash. They just go in and book them all at once, I never have a chance. 

Can we have some rule that one cannot book more than one week ahead or something like that?

**Comment by RasmusRossetti** [comment]
Hi Joel,

I understand that you're frustrated with another person always booking the Sunday slots from 15-18, making it difficult for you to find a time to wash.

While it may be tempting to restrict bookings to one week ahead, the purpose of having a timelock is to allow people to book in advance and have some flexibility with their schedules. However, we have implemented some restrictions to ensure that everyone has a fair chance to book slots.

For example, we have limited the number of times someone can book a slot to just two times, and we have also restricted users to booking only one timeslot per day. These measures should help prevent one person from monopolizing the booking system and give others a fair chance to book the slots they need.

We hope that these measures will help alleviate some of the frustration you've been experiencing, and we appreciate your patience as we work to improve the system.

Best,
The Team
